### DISCUSSION TITLE
RemoteConfig setMinimumFetchInterval for iOS and Android

### DISCUSSION THREAD

**Discussion by sagaraya** [capawesome-team/capacitor-firebase#685] [Q&A] [answered]
Why is the setMinimumFetchInterval method not implemented in the RemoteConfig package for iOS/Android?

The code is written as `call.reject("Not available on iOS.")`, but it is defined in the Firebase API reference.

* iOS: https://firebase.google.com/docs/reference/swift/firebaseremoteconfig/api/reference/Classes/RemoteConfigSettings?_gl=1*ptpwho*_up*MQ..*_ga*MTI0MDI0NTY2NS4xNzIyMzA0OTk3*_ga_CW55HF8NVT*MTcyMjMwNDk5Ny4xLjAuMTcyMjMwNTA3MC4wLjAuMA..
* Android: https://firebase.google.com/docs/reference/android/com/google/firebase/remoteconfig/FirebaseRemoteConfigSettings.Builder?_gl=1*fqm33u*_up*MQ..*_ga*MTE4MDA5Njg5OS4xNzIyMzA2MTg3*_ga_CW55HF8NVT*MTcyMjMwNjE4Ny4xLjAuMTcyMjMwNjE5NC4wLjAuMA..#setMinimumFetchIntervalInSeconds(long)

**Comment by robingenz** [comment, marked as answer]
I can't remember why I didn't add them. These APIs probably didn't exist back then. Feel free to create a feature request.
